addons/wowl/static/src/services/model_service.js

```diff
@@ -40,7 +41,14 @@ function readGroup(rpc, env, model) {
     if (options.limit) {
       kwargs.limit = options.limit;
     }
-    return callModel(rpc, env, model)("web_read_group", [], kwargs);
+    const result = await callModel(rpc, env, model)("web_read_group", [], kwargs);
+    if (result && result.length === 1) {
+      // hum, no result really?
+      // API correction of read_group
+      result.groups[0].__domain = result.groups[0].__domain || [];
```

Comment:
to do? or make another read_group function that makes some processing for the reporting view?

Reply:
not to do!

Reply:
create other function --> find name + implementation